pkg/client/client.go

```diff
@@ -68,10 +68,11 @@ import (
 )
 
 const (
-	deploymentCreateTimeout = 5 * time.Minute
-	deploymentDeleteTimeout = 5 * time.Minute
-	metadataPrefix          = "monitoring.openshift.io/"
-	clusterConsole          = "cluster"
+	deploymentCreateTimeout       = 5 * time.Minute
+	deploymentDeleteTimeout       = 5 * time.Minute
+	metadataPrefix                = "monitoring.openshift.io/"
+	clusterConsole                = "cluster"
+	VerticalPodAutoscalerGRString = "verticalpodautoscalers.autoscaling.k8s.io"
```

Comment:
```suggestion
	VerticalPodAutoscalerCRString = "verticalpodautoscalers.autoscaling.k8s.io"
```
Just wondering if the name is supposed to be VerticalPodAutoscaler**GR**String. Feel free to ignore.

Reply:
I suffixed it GR to reflect the "resource.group" format of that string (analogous to GVK). Would something like `VerticalPodAutoscalerCRDMetadataName` (or `VerticalPodAutoscalerGroupResourceString`) be a better candidate instead?

Reply:
That's what I came up with too...so that should work. We can always change the name in a follow up though.